src/components/AsyncPage.tsx

```diff
@@ -10,7 +10,7 @@ type AsyncPageProps = {
     fetchStatus: AsyncStatus;
     errorMessage: string;
     emptyContentMessage?: string;
-    children: JSX.Element;
+    children: JSX.Element | JSX.Element[];
```

Comment:
Is this in preparation for something else? I can't find anywhere where the asyncPage has more than one child